import-dashboards.py

```diff
@@ -242,6 +242,25 @@ def add_datasources(api_key):
             print ' * Cannot add PTSummary Data Source'
             sys.exit(-1)
 
+    if 'Prometheus AlertManager' not in ds:
+        print ' * Prometheus AlertManager Data Source'
+        data = json.dumps({
+            'name': 'Prometheus AlertManager',
+            'type': 'camptocamp-prometheus-alertmanager-datasource',
+            'url': 'http://localhost/alertmanager',
```

Comment:
https://github.com/percona/pmm-server/pull/229/files#r494947820

Reply:
Are you suggesting to use host:port to communicate with alertmanager directly?

Reply:
Yes, like we do for PostgreSQL and ClickHouse above

Reply:
Done